Brain/brain_constants.h

```diff
@@ -0,0 +1,52 @@
+#include <TimeLib.h>
+#include <ducer.h>
+
+/*
+ * Data structure to store all information relevant to a specific sensor type.
+ */
+struct sensorInfo {
+  String sensor_name;
+  int board_address;
+  int sensor_id;
+  int overall_id;
+  int clock_freq;
+  void (*dataReadFunc)(float *data);
+  // sensorInfo(String n, int b, byte s, int o, int f) : sensor_name(n), board_address(b), sensor_id(s), overall_id(o), clock_freq(f) {}
+};
+
+/*
+ * Data structure to store all information relevant to a specific valve.
+ */
+struct valveInfo {
+  String name;
+  int id;
+  int (*openValve)();
+  int (*closeValve)();
+};
```

Comment:
I think these should still be in utils, since they are structs, not really constants. This constants file should only contain things that might change (pin numbers, num sensors, array of sensors, etc).

Reply:
Yea but the tricky thing is then I can't import brain utils in both brain constants and brain.ino because it says the variables were initialized twice. I think can use the extern keyword though. Will check this out